saleor/graphql/warehouse/types.py

```diff
@@ -71,6 +69,10 @@ def resolve_shipping_zones(root, *_args, **_kwargs):
         ]
         return shipping_zones
 
+    @staticmethod
+    def resolve_company_name(root, *_args, **_kwargs):
+        return root.address.company_name
```

Comment:
⚠️ Possible performance issue ⚠️ 

This will perform extra SQL `SELECT ... FROM ...address WHERE id=n` on each warehouse entry and I see we use this type in some parent types, should utilize a dataloader.

Reply:
We should use "AddressByIdLoader".

Reply:
good catch, thanks. Done.